### PR TITLE
Change development environment to Gradle and Android Studio

### PR DESCRIPTION
**Pull request by miguelcardo**
So that it can be built using command-line tools. Plus Android Studio is now the Google-recommended development environment for Android, and runs faster than Eclipse.

### CODE REVIEW

**Comment by miguelcardo** [comment]
@petter-fidesmo @slomo Ready for review!
I'll rename the repo once this is merged

File: README.md [hunk 1]
```diff
@@ -1,25 +1,40 @@
...
-- In Eclipse (with the [ADT plugin](http://developer.android.com/tools/sdk/eclipse-adt.html)), select menu *File -> Import*
-- In the dialogue box, select *General -> Existing Projects into Workspace*
-- Click *Next*
-- As root directory, select the one where you have cloned this repository
-
-Screenshot
----------
-![Screenshot showing the two buttons and an example UUID](/docs/Example_screenshot.png "Screenshot showing the two buttons and an example UUID")
+- In the project's root directory, type ``./gradlew build``
+- Copy the Android app's APK from ``build/apk/android-transceive-demo-release-unsigned.apk`` to your phone and install it manually.
```
Comment (slomo) [inline README.md]: I installed with gradel via `./gradlew installDebug` I also tried `adb install  build/apk/android-transceive-demo-release-unsigned.apk` but it failed with `Failure [INSTALL_PARSE_FAILED_NO_CERTIFICATES]`

**Comment by slomo** [comment]
Works for me so :+1:

**Pull request merged**